Functions.Templates/Templates/RabbitMQTrigger-CSharp/.template.config/template.json

```diff
@@ -51,7 +51,8 @@
             "ManualInstructions": [],
             "args": {
                 "referenceType": "package",
-            "reference": "Microsoft.Azure.WebJobs.Extensions.RabbitMQ", "version": "1.1.0",
+                "reference": "Microsoft.Azure.WebJobs.Extensions.RabbitMQ",
+                "version": "2.0.3",
```

Comment:
Are we comfortable with major version update here?

Reply:
Validating nothing is broken here, should know soon!

Reply:
Can we leave this one unchanged? Any drawbacks of doing that?

Reply:
No issues with the template, works as expected